### PR TITLE
[EXAMPLE] Add a JavaScript project bundled with webpack

### LINKED ISSUE #68: [FEAT] Add an example for integration in a Vanilla Javascript project
Same as for #19 
For now we only have examples to be used directly in the browser

We may use
- webpack as we already used rollup in #19 
- the commonjs bundle as we already provided an example with the ESM bundle in #19. This requires https://github.com/process-analytics/bpmn-visualization-js/issues/595

### PR DESCRIPTION
**Pull request by tbouffard**
Demonstrate how to integrate `bpmn-visualization` into a webpack bundled project,
providing tips for webpack configuration.
Load a BPMN diagram and displays an overlay on a BPMN activity.

closes #68

### CODE REVIEW

File: examples/projects/javascript-vanilla-with-webpack/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Usage
+
+To run locally:
+
+1. `npm install`
+2. `npm run serve`
+3. [localhost app](http://localhost:8080)
+
+You should see a BPMN diagram displayed in the page
```
Comment (csouchet) [inline examples/projects/javascript-vanilla-with-webpack/README.md:11]:
```suggestion
You should see a BPMN diagram displayed in the page.
```

File: examples/projects/javascript-vanilla-with-webpack/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+![BPMN diagram in the home page](docs/home.png)
```
Comment (csouchet) [inline examples/projects/javascript-vanilla-with-webpack/README.md:13]: Is it the end of the previous sentence ?

Comment (tbouffard) [reply examples/projects/javascript-vanilla-with-webpack/README.md:13]: Not really, I am going to rewrite it 😸

Comment (tbouffard) [reply examples/projects/javascript-vanilla-with-webpack/README.md:13]: Done in f5d24f6

**Review by csouchet — APPROVED** [review]

**Pull request merged**